src/router/openai_proxy.py

```
@@ -0,0 +1,56 @@
+async def proxy_chat_completions(
+    request: Request,
+    x_request_id: Optional[str] = Header(None),
+    authorization: Optional[str] = Header(None),
+):
+    body = await request.body()
+
+    req_id = x_request_id or str(uuid.uuid4())
+
+    try:
+        ChatRequest.model_validate_json(body)
+    except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)